### ISSUE TITLE
microbenchmark: debug double client merge

### ISSUE THREAD

**Issue by michaelsevilla** [michaelsevilla/cudele-popper#14]
A single client decoupling the namespace and merging updates later works fine (it overwrites the results from `mdtest` but this is fine because the generated load is the same). With multiple clients (each writing to their own directories), the merge fails because the metadata server cannot write out the inodes. It complains that it cannot pull inodes from the free inode pool.

- [ ] single node debugging; spin up two clients and get them to merge one after the other
- [ ] figure out what the free range of the pool is
- [ ] walk through the inodes that are trying to get allocated and add debugging that spits out what the range of the free inode pool is

**Comment by michaelsevilla** [comment]
Got this to work single node. Now there is something wrong with the Ansible code. Check the start inodes and decoupling directory code in `~/msevilla/cudele-popper/experiments/fuse/scale-merge/run.sh`. The following sequences work when going from one node:

```bash
docker exec ceph-issdm-12-mds ceph daemon mds.issdm-12 decouple bogus0 0 100
docker exec ceph-issdm-12-mds cephfs-journal-tool event create list --nfiles 10 --persist true --decoupled_dir bogus0 --file /etc/ceph/test.bin --start_ino  1099511628877
docker exec ceph-issdm-12-mds ceph daemon mds.issdm-12 merge /etc/ceph/test.bin
docker exec ceph-issdm-12-mds ceph daemon mds.issdm-12 decouple bogus1 0 100
docker exec ceph-issdm-12-mds cephfs-journal-tool event create list --nfiles 10 --persist true --decoupled_dir bogus1 --file /etc/ceph/test.bin --start_ino 1099511628977
docker exec ceph-issdm-12-mds ceph daemon mds.issdm-12 merge /etc/ceph/test.bin
```

Perhaps it only works from single node -- this is fine because we can simulate creates everywhere with a giant merge step at the end (pretend that all the files are already on the metadata server).

**Comment by michaelsevilla** [comment]
Fixed on 8adf01b98af39502aa7fe3d10c4b19935508eaa7.
